### PR TITLE
feat: multilingual search

### PR DESCRIPTION
**Pull request by thessakockelkorn**

### CODE REVIEW

File: src/Services/Indexer/IndexBuilderService.php [hunk 1]
```diff
@@ -28,7 +26,7 @@ public function __construct(string $serverType, $debug = false)
 
         $locale = env('SB_LOCALES_DEFAULT', 'nl');
```
Comment (64knl) [inline src/Services/Indexer/IndexBuilderService.php:27]: This is the BACKEND default locale. Not the site default locale,

File: src/Services/Indexer/IndexBuilderService.php [hunk 2]
```diff
@@ -28,7 +26,7 @@ public function __construct(string $serverType, $debug = false)
...
         $this->defaultLocale = Lang::where('url', $locale)->get();
```
Comment (64knl) [inline src/Services/Indexer/IndexBuilderService.php:28]: This should probably be Lang::default()

**Review by 64knl — CHANGES_REQUESTED** [review]
Default language implementation seems incorrect to me.

File: src/Services/Indexer/IndexBuilderService.php (hunk 1 above)
Comment (thessakockelkorn) [reply src/Services/Indexer/IndexBuilderService.php]: As $this->defaultLocale was not used I removed both lines.

**Review by 64knl — APPROVED** [review]

**Pull request merged**